service/notifications.go

```diff
@@ -93,7 +95,8 @@ func (producer *AvailabilityStatusNotifier) EmitAvailabilityStatusNotification(a
 		Application: application,
 		EventType:   statusEventType,
 		Timestamp:   time.Now().Format(time.RFC3339),
-		AccountID:   accountNumber,
+		AccountID:   id.AccountNumber,
+		OrgId:       id.OrgID,
```

Comment:
I think it is not allowed to have org id here: https://core-platform-apps.pages.redhat.com/notifications-docs/dev/user-guide/send-notification.html for now. I will try to get more info.

Reply:
I see. I've removed the "OrgId" from the payload and kept only the "AccountNumber"

Reply:
Currently they accept it now: https://coreos.slack.com/archives/C01UC7NRV19/p1652199785586619 but they ignore it - so we can leave it as you have in last change.

Reply:
All right, I've reverted the code so that it includes the "OrgId" now as well.